tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,65 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(5)
+  require(shinytest2)
+
+  data <- simple_cdisc_data()
+  data <- within(data, {
+    attr(ADSL, "metadata") <- list("Author" = "NEST team", "data_source" = "synthetic data")
+  })
+
+  app <- TealAppDriver$new(
+    data = data,
+    modules = teal::modules(
+      tm_front_page(
+        header_text = c(
+          "Important information" = "It can go here.",
+          "Other information" = "Can go here."
+        ),
+        tables = list("MTCARS" = head(mtcars, 5), "IRIS" = head(iris, 5)),
+        additional_tags = HTML("Additional HTML or shiny tags go here <br>"),
+        footnotes = c("X" = "is the first footnote", "Y is the second footnote"),
+        show_metadata = TRUE
+      )
+    ),
+    timeout = 3000
+  )
+
+  app$wait_for_idle()
```

Comment:
This is not necessary due to https://github.com/insightsengineering/teal/blob/05c0775fa56f85496f582fc8dc16bc506e88c89c/R/TealAppDriver.R#L69

```suggestion
```